map_gen/maps/grid_islands_rotated.lua

```diff
@@ -177,4 +197,14 @@ map = b.any {map, paths, sea}
 map = b.change_map_gen_collision_tile(map, 'water-tile', 'grass-1')
 map = b.rotate(map,degrees(45))
 
+
+-- Disable landfill technology
+local function on_init()
+    local player_force = game.forces.player
+    player_force.technologies['landfill'].enabled = false           -- disable landfill
```

Comment:
You removed this part:
> -- For best balance run the following commands after map generation:
> -- /silent-command game.forces["player"].technologies["landfill"].enabled = false
> -- /silent-command game.forces.player.character_running_speed_modifier = 1.5
> -- /silent-command game.difficulty_settings.technology_price_multiplier=2

Should we include it in the on_init() function?

```suggestion
    player_force.technologies['landfill'].enabled = false           -- disable landfill
    player_force.character_running_speed_modifier = 1.5
    game.difficulty_settings.technology_price_multiplier=2
```

Reply:
I think I tested that and it didn't work.  I didn't have time to investigate why.

Reply:
I just double checked them, and they should work

Reply:
No idea what I was doing wrong then. If you've tested the changes then they should be committed.